tests/unit/test_solvers.py

```diff
@@ -923,9 +920,10 @@ def test_map_reduce_analytic(self):
 
         Concatenating we obtain
         :math:`[0, 30, 0, 40, 90, 50, 100, 150, 110, 160, 210, 170]`.
-        Now we repeat the same process, we check how many partitions
-        (has to be power of 2) we want to divide this new data of size 12 into,
-        our new options for partitioning are:
+        We repeat the process, checking how many partitions we want
+         to divide this intermediate dataset (of size 12) into.
+        Recall, this number of partitions must be a power of 2.
+        Our options are::
```

Comment:
Reduce to 3 lines, and delete duplicated colon

    We repeat the process, checking how many partitions we want to divide this
    intermediate dataset (of size 12) into. Recall, this number of partitions must
    be a power of 2. Our options are: